src/main/java/com/capitalone/dashboard/service/TestResultServiceImpl.java

```diff
@@ -92,25 +110,25 @@ private void addAllTestResultRepositories(TestResultRequest request, List<TestRe
         }
     }
 
-    private void validateTestCapabilities(TestResultRequest request, QTestResult testResult, BooleanBuilder builder) {
+    private void validateTestCapabilities(com.capitalone.dashboard.request.TestResultRequest request, QTestResult testResult, BooleanBuilder builder) {
         if (!request.getTypes().isEmpty()) {
             builder.and(testResult.testCapabilities.any().type.in(request.getTypes()));
         }
     }
 
-    private void validateDurationRange(TestResultRequest request, QTestResult testResult, BooleanBuilder builder) {
+    private void validateDurationRange(com.capitalone.dashboard.request.TestResultRequest request, QTestResult testResult, BooleanBuilder builder) {
```

Comment:
Can you fix the import so that package name does not appear in thee method signature.